php/Terminus/Collections/Workflows.php

```diff
@@ -1,117 +1,59 @@
 <?php
 
-namespace Terminus\Models\Collections;
+namespace Terminus\Collections;
 
 use Terminus\Session;
-use Terminus\Models\Workflow;
 
 class Workflows extends TerminusCollection {
   /**
-   * @var string
+   * @var Environment
    */
-  protected $environment;
+  private $environment;
   /**
-   * @var object
-   * @todo Find specific type for this
+   * @var Organization
    */
-  protected $owner;
-
+  private $organization;
   /**
-   * Creates a new workflow and adds its data to the collection
-   *
-   * @param string $type    Type of workflow to create
-   * @param array  $options Additional information for the request, with the
-   *   following possible keys:
-   *   - environment: string
-   *   - params: associative array of parameters for the request
-   * @return Workflow $model
+   * @var Site
    */
-  public function create($type, array $options = array()) {
-    $options = array_merge(array('params' => array()), $options);
-    if (isset($options['environment'])) {
-      $this->environment = $options['environment'];
-    }
-    $params = array_merge($this->args, $options['params']);
-
-    $results = $this->request->request(
-      $this->getFetchUrl(),
-      array(
-        'method'      => 'post',
-        'form_params' => array(
-          'type'   => $type,
-          'params' => (object)$params
-        )
-      )
-    );
-
-    $model = new Workflow(
-      $results['data'],
-      array(
-        'owner' => $this->owner,
-      )
-    );
-    $this->add($model);
-    return $model;
-  }
-
+  private $site;
   /**
-   * Give the URL for collection data fetching
-   *
-   * @return string URL to use in fetch query
+   * @var User
    */
-  protected function getFetchUrl() {
-    $url = '';
-    switch ($this->getOwnerName()) {
-      case 'user':
-        $url = sprintf(
-          'users/%s/workflows',
-          $this->owner->id
-        );
-          break;
-      case 'site':
-        $replacement = $this->owner->get('id');
-        if (isset($this->environment)) {
-          $replacement = sprintf(
-            '%s/environments/%s',
-            $this->owner->get('id'),
-            $this->environment
-          );
-        }
-        $url = sprintf(
-          'sites/%s/workflows',
-          $replacement
-        );
-          break;
-      case 'organization':
-        $user = Session::getUser();
-        $url = sprintf(
-          'users/%s/organizations/%s/workflows',
-          $user->id,
-          $this->owner->get('id')
-        );
-          break;
-    }
-    return $url;
-  }
+  private $user;
+  /**
+   * @var string
+   */
+  protected $collected_class = 'Terminus\Models\Workflow';
 
   /**
-   * Fetches workflow data hydrated with operations
+   * Instantiates the collection, sets param members as properties
    *
-   * @param array $options Additional information for the request
-   * @return void
+   * @param array $options To be set to $this->key
```

Comment:
This isn't true anymore. This should say
`@param array $options Options with which to configure this collection`